apps/civicsignalblog/src/payload/collections/Research/Tags.js

```diff
@@ -1,5 +1,5 @@
-import canRead from "../../access/applications/research";
-import slug from "../../fields/slug";
+import canRead from "#civicsignalblog/payload/access/applications/research";
+import slug from "#civicsignalblog/payload/fields/slug/index";
```

Comment:
Doesn't it work without specifying `index`?

Reply:
@kilemensi, currently not working I'm looking and the package.json and tsconfig.json configs...should've been straight forward but its not resolving automatically...this is the major blocker, same as explained [here](https://github.com/CodeForAfrica/ui/pull/902?notification_referrer_id=NT_kwDOAGhUF7MxMjYwNTM0Njg5OTo2ODM3Mjcx#issuecomment-2382555427)

Reply:
@kilemensi, I have done a lot of digging around in this one, there is no clean way (atleast I can think of) to escape using index when we are importing...the limitations I am facing is that with subpath imports there is no fallback mechanism, It would've been ideal if there was fallback mechanism when we are resolving the paths in the array of URLs we have configured 
```
[
    "./src/*/index.js",
    "./src/*.js",
    "./src/*"
]
```
However, on first match of the string pattern if no file is found it will throw an error.

One way I have tested that could work is by explicitly configuring the full path in the package.json to match like this 👇 
```
    "#civicsignalblog/payload/fields/slug": {
      "dev": [
        "./src/payload/fields/slug/index.js"
      ],
      "default": [
        "./src/payload/fields/slug/index.js"
      ]
    },

```
But this seems a bit too much, for the problem we are trying to solve...I'm open to suggestions if we should merge it as is for now

Reply:
I'll create this as a separate issue we can explore later !